manifest.json

```diff
@@ -1,9 +1,10 @@
 {
+    "manifest_version": 2,
     "name": "Casey's Extension",
-    "version": "1",
+    "version": "1.1.0",
```

Comment:
Here, we wanna just go with "1.0.0". We only increment the version when there's a public release - not as part of the process of creating and editing code in preparation for one. For the purposes of this project, we will have just the one 'release' at the end, when the vertical slice is done.